CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,64 @@
+# 1Password Community Code of Conduct
+The Passage by 1Password GitHub exists to facilitate information sharing and deliver resources to developers who are interested in Passage products.
```

Comment:
I refer to this as the "Passage by 1Password GitHub" here and shorten to "Passage GitHub" for the rest of the document. I can swap to the longer one throughout if we feel that is better

Reply:
I think this is fine. For docs v2 the guidance was use full terminology for products at the top-level/first mention (eg. Passage Passkey Flex vs. Passkey Flex) and go shorter afterwards, this follows the same convention.

Reply:
Yep exactly, that's why I made this choice